modules/ROOT/pages/about-platform-apis.adoc

```diff
@@ -1,4 +1,4 @@
-= About Exchange Platform APIs
+= Use Exchange Platform APIs
```

Comment:
Topics that began with "About" were not task topics, but concept topics. Because concept topics do not include tasks, It doesn't make sense to place a verb at the beginning of the title of a concept topic.

I see similar changes to many topics that originally began with "About". Yes, the guideline that I pointed you to yesterday said to start titles with verbs in the imperative mood, rather than with gerunds. However, only the titles of task topics start with verbs.